src/web/app/pages-student/student-course-details-page/student-course-details-page.component.ts

```diff
@@ -115,26 +119,27 @@ export class StudentCourseDetailsPageComponent implements OnInit {
           }
 
           this.studentProfileService.getStudentProfile(student.email, courseId)
-                .subscribe((studentProfile: StudentProfile) => {
-                  const newPhotoUrl: string =
-                    `${environment.backendUrl}/webapi/student/profilePic`
-                    + `?courseid=${courseId}&studentemail=${student.email}`;
-
-                  const newTeammateProfile: StudentProfileWithPicture = {
-                    studentProfile: {
-                      ...studentProfile,
-                      email: student.email,
-                      shortName: student.name,
-                    },
-                    photoUrl : newPhotoUrl,
-                  };
-
-                  this.teammateProfiles.push(newTeammateProfile);
-                });
```

Comment:
Same comment regarding unnecessary indentation change

Reply:
I think the first one is more understandable for anyone new on this project and who want to work on it

Reply:
@hamza67100 the issue is not about "more understandable" or not (which in itself is debatable). The issue is you're introducing changes which are not related to the issue at hand here.